docs/deployment/k4k8s-enterprise.md

```diff
@@ -148,7 +148,7 @@ $ helm install kong/kong \
 
 # Helm 3
 $ helm install kong/kong --generate-name
-    --name demo --namespace kong \
+    demo --namespace kong \
```

Comment:
Should `demo` be removed as well?
`--generate-name` actually generates a name and doesn't take in a parameter.
https://github.com/helm/helm/blob/bf9c64f48ee8f799c89de6cfbd44302f798b3531/cmd/helm/install.go#L143

Reply:
It should.